indico/modules/events/sessions/controllers/management/sessions.py

```diff
@@ -0,0 +1,119 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from flask import request, redirect
+
+from indico.core.db import db
+from indico.core.db.sqlalchemy.colors import ColorTuple
+from indico.modules.events.sessions.controllers.management import RHManageSessionsBase, RHManageSessionBase
+from indico.modules.events.sessions.forms import CreateSessionForm
+from indico.modules.events.sessions.views import WPManageSessions
+from indico.modules.events.sessions.util import create_session
+from indico.web.flask.util import url_for
+from indico.web.forms.base import FormDefaults
+from indico.web.util import jsonify_data, jsonify_template
+
+
+class RHSessionsList(RHManageSessionsBase):
+    """Display list of all sessions within the event"""
+
+    def _process(self):
+        active_sessions = self.event.sessions.filter_by(is_deleted=False).all()
+        return WPManageSessions.render_template('management/sessions/sessions_list.html', self.event.as_legacy,
+                                                event=self.event, sessions=active_sessions)
+
+
+class RHAddSession(RHManageSessionsBase):
+    """Add a session to the event"""
+
+    def _process(self):
+        form = CreateSessionForm()
+        if form.validate_on_submit():
+            session_config = dict(is_poster=(form.session_type.data == 'poster'))
```

Comment:
add a `generated_data` item to the form instead so you have `is_session` in there.